autobotAI_integrations/integrations/azure_devops/code_sample.py

```
@@ -0,0 +1,66 @@
+                if repos:
+                    for repo in repos:
+                        project_data["repositories"].append(
+                            {
+                                "repo_id": repo.id,
+                                "repo_name": repo.name,
+                                "default_branch": repo.default_branch,
+                                "web_url": repo.web_url,
+                                "size": repo.size,
+                            }
+                        )
+            except Exception as e:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)